### PR TITLE
Document the new `--fallback-extensions` option

### PR DESCRIPTION
**Pull request by dscho**
This is a companion to https://github.com/lycheeverse/lychee/pull/1422, ready to go once that PR is merged and a new version is published.

### CODE REVIEW

File: src/content/docs/recipes/pretty-urls.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+to https://lychee.cli.rs/index or to https://lychee.cli.rs/index.html, either URL
+will resolve to the same page.
+
+To help with locally checking links in such scenarios, where the links lack the
+`.html` extension but the files on disk have them, Lychee supports the option
+`--fallback-extensions`:
+
+```bash
+# If a local link without a file extension does not resolve to an existing file or
+# directory, let Lychee fall back to try again after appending the `.html` or the
```
Comment (mre) [inline src/content/docs/recipes/pretty-urls.md:20]:
```suggestion
# directory, let lychee fall back to try again after appending the `.html` or the
```

**Comment by mre** [comment]
Looks good. Just a minor typo.

**Comment by dscho** [comment]
> Looks good. Just a minor typo.

@mre I committed as-is; Would you prefer the fixup to be squashed into the faulty commit instead?

**Comment by mre** [comment]
Either way is fine with me. I can squash the commits on merge. Can you resolve the merge conflict (which I caused, with https://github.com/lycheeverse/lycheeverse.github.io/commit/9fe5aa7c3f0c2650091b7303c884537d414908ce)?

**Comment by dscho** [comment]
> Can you resolve the merge conflict (which I caused, with [9fe5aa7](https://github.com/lycheeverse/lycheeverse.github.io/commit/9fe5aa7c3f0c2650091b7303c884537d414908ce))?

Done! While at it, I also squashed the spell fix in.

**Pull request merged**